diesel/src/query_dsl/mod.rs

```diff
@@ -477,6 +477,55 @@ pub trait QueryDsl: Sized {
         methods::FilterDsl::filter(self, predicate)
     }
 
+    /// Adds to the `WHERE` clause of a query using `OR`
+    ///
+    /// If there is already a `WHERE` clause, the result will be `(old AND new)`.
```

Comment:
I might be misunderstanding this but should this be `(old OR new)`?

Reply:
Good catch